pkg/projects/descriptions.json

```diff
@@ -20,8 +25,23 @@
         "CommonTypes": [ ]
     },
     {
+        "Name": "Microsoft.NETCore",
+        "Description": "Provides a set of packages that can be used when building portable libraries on .NETCore based platforms.",
+        "CommonTypes": [ ]
+    },
+    {
         "Name": "Microsoft.NETCore.App",
         "Description": "A set of .NET API's that are included in the default .NET Core application model.",
         "CommonTypes": [ ]
     },
+    {
+        "Name": "Microsoft.NETCore.Console",
```

Comment:
Is there any relationship (from composition standpoint) between this and NETCore.App? Likewise for .NETCore/.NETCoreApp/.NETCore.Console?